include/config.h

```diff
@@ -43,6 +43,10 @@
 #error "Missing or unrecognized game language define"
 #endif
 
+#ifndef __MWERKS__
+#define NONMATCHING
```

Comment:
????
how does this.... fix anything?

Reply:
In editors, and when building under gcc or clang, this makes the nonmatching code active instead of the inline asm. Mainly an aesthetic help.

Reply:
no yeah I figured, but why move it?

Reply:
Moved it here so you can import `NONMATCHING` into assembly files via `global.inc` (e.g. `launch_application_data.s`). Importing all of `global.h` causes an assembler error.

Reply:
fair